sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -141,12 +140,13 @@ Make sure the client-id can access target Key Vault. Here are steps to configure
 
 #### Using a managed identity
 
-If you are using managed identity instead of client-id, add these items in your `application.yml`:
+If you are using managed identity instead of App registrations, add these items in your `application.yml`:
 
 ```yaml
 azure:
   keyvault:
     uri: <the URI of the Azure Key Vault to use>
+    managed-identity: # client-id of the user-assigned managed identity to use. If empty, then system-assigned managed identity will be used.
```

Comment:
Should we comment this line? I mean use `system-assigned managed identity` as default value.